spec/views/users/login.html.erb_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.feature 'Login page', type: :feature do
+  background { visit new_user_session_path }
+  scenario 'displays form fields' do
+    expect(page).to have_field('user_email')
+    expect(page).to have_field('user_password')
+    expect(page).to have_button('Log in')
+  end
+
+  describe 'login page' do
+    scenario 'If the form submitted is empty' do
+      click_button 'Log in'
+      expect(page).to have_content "Forgot your password? Didn't receive confirmation instructions?"
+    end
+
+    scenario 'Submitting form with the incorrect email and password' do
+      visit new_user_session_path
+      fill_in 'Email', with: 'david@gmail.com'
+      fill_in 'Password', with: ''
+      click_button 'Log in'
+      expect(page).to have_content "Forgot your password? Didn't receive confirmation instructions?"
+    end
+
+    scenario 'Submitting form with the correct email and password' do
+      @user = User.create(name: 'daudi kabaka', email: 'david@gmail.com', password: '123456', bio: 'Iam a software developer',
+                          photo: 'https://david.com', posts_counter: 0)
+      visit new_user_session_path
+      fill_in 'Email', with: @user.email
+      fill_in 'Password', with: @user.password
+      click_button 'Log in'
```

Comment:
- [OPTIONAL] Please for the test to complete, it will be nice to add your expectation, what is your expectation after clicking on the on the `Login button`

Reply:
I'll look into it. Thank you for taking the time to review my code.